packages/venia-ui/lib/components/CreateAccount/createAccount.js

```diff
@@ -82,10 +84,7 @@ class CreateAccount extends Component {
                 initialValues={initialValues}
                 onSubmit={handleSubmit}
             >
-                <h3 className={classes.lead}>
-                    {`Check out faster, use multiple addresses, track
-                         orders and more by creating an account!`}
-                </h3>
+                <h3 className={classes.lead}>{LEAD}</h3>
```

Comment:
We probably don't need this to be a header anymore especially since we set line and font size. I would reserve it for a11y purposes.

Reply:
Heading. 😉 

I agree, though; semantically this should just be a paragraph.

Reply:
You're heading for trouble!

Cool thanks.

Reply:
This has been addressed.